assemblies/assembly-openshift-operator.adoc

```diff
@@ -25,6 +25,9 @@ ifdef::PAM[]
 ** For a production or immutable environment, 2 gigabytes of memory and 1 virtual CPU core for each replica of the {CENTRAL} Monitoring pod.
 endif::PAM[]
 ** 2 gigabytes of memory and 1 virtual CPU core for each replica of each {KIE_SERVER} pod.
+ifdef::PAM[]
+** 1 gigabyte of memory and half a virtual CPU core for each replica of a Smart Router pod.
```

Comment:
Question - Is it possible to assign half a virtual CPU core?

Reply:
Yup, it's called 500m or something